akka-docs/src/main/paradox/typed/persistence.md

```diff
@@ -462,22 +462,22 @@ Retention of snapshots and events are controlled by a few factors. Deletes to fr
 
 To free up space, an event sourced actor can automatically delete older snapshots 
 based on a user provided or default `RetentionCriteria` @scala[from `withRetention`] @java[by overriding `retentionCriteria`]
-combined with either the `snapshotWhen` or `snapshotEvery` methods. 
+combined with the `snapshotWhen` method. 
```

Comment:
We should hold off on this now and reconsider in the related https://github.com/akka/akka/issues/26545